doc/release-notes.md

```diff
@@ -335,6 +335,10 @@ RPC
 Tests
 -----
 
+- The BIP 325 default signet can be enabled by the `-chain=signet` or `-signet`
+  setting. The settings `-signetchallenge` and `-signetseednode` allow
+  modifying the network.
```

Comment:
"modifying the network" -> "enabling a custom signet instead" ?

Reply:
thx done